src/puppetlabs/puppetdb/cli/services.clj

```diff
@@ -54,7 +53,8 @@
             [puppetlabs.puppetdb.command.dlo :as dlo]
             [puppetlabs.puppetdb.config :as conf]
             [puppetlabs.puppetdb.http.server :as server]
-            [puppetlabs.puppetdb.jdbc :as pl-jdbc :refer [with-transacted-connection]]
+            [puppetlabs.puppetdb.jdbc :as jdbc
+             :refer [with-transacted-connection]]
```

Comment:
might as well move calls to `with-transacted-connection` under the `jdbc` ns as well